src/components/WalletUnlock/DeterministicAccountTable.tsx

```diff
@@ -372,7 +372,7 @@ const DeterministicTable = ({
             onClick={generateFreshAddress}
             disabled={!isComplete || freshAddressIndex >= DEFAULT_GAP_TO_SCAN_FOR}
           >
-            <Icon type="add" color="none" />
+            <Icon type="add" color="none" width="32px" />
```

Comment:
This is not the same size as the blockies, is that intentional?

Reply:
I just removed the sizing on the svg, I put the value that was previously set in the svg's code. It was this size before :)

Reply:
I saw, just confirming that this is intentional.

Reply:
Not really part of the work but I initially wanted to use the `add` icon - it was slightly different from the one I needed so I left the removal of the width/height but didn't use the icon :)